app/templates/road-sign-concepts/road-sign-concept.hbs

```diff
@@ -1,3 +1,5 @@
+{{page-title (t "roadsignconcept.details")}}
```

Comment:
I think it might be better to display the roadSignConceptCode instead? Similar to the breadcrumb?

Reply:
I was thinking about that for the road signs only, but I preferred the current solution in order to have a "common" interface between road signs and measures (in which use the breadcrumb content is inapplicable because is too long).
Of course I'm opened to further discussion.